doc/api/cli.md

```diff
@@ -51,6 +51,53 @@ entry point, the `node` command will accept as input only files with `.js`,
 [`--experimental-wasm-modules`][] is enabled; and with no extension when
 [`--experimental-default-type=module`][] is passed.
 
+## Sub-commands
+
+### `node run <command> [<arguments>]`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+This runs a specified command from a package.json's `"scripts"` object.
+If no `"command"` is provided, it will list the available scripts.
+
+`node run` prepends `./node_modules/.bin`, relative to the current
```

Comment:
One of the options for the "version manager" discussion which I believe should be kept on the table is to use `node_modules/.bin` as a way to manage the users `PATH` for tool versions their projects depend on. If we land this PR, that could mean that something like `node run node` is part of the platform experience. Since I am being asked to rush in feedback on this I just wanted to point out this might be problematic or at least confusing and I am not sure I have a clear answer about how this might play together.

Reply:
This is experimental, so we can always change it as needed (or remove it).